test-data/unit/check-protocols.test

```diff
@@ -1079,6 +1079,44 @@ class C2(P):
 C()
 C2()
 
+[case testCannotInstantiateEmptyMethodExplicitProtocolSubtypes]
```

Comment:
Since we differentiate `.py` and `.pyi` files, we 100% need to test it.

Reply:
Right, that's a good point.